Commands.md

```diff
@@ -1544,3 +1546,11 @@ total 308
 ## git abort
 
 Abort current rebase, merge or cherry-pick, without the need to find exact command in history.
+
+## git auto
+
+Commits staged changes with a generated commit message summarizing changes. (Allows edit)
+
+## git magic
```

Comment:
Could we merge these two commands into one?

Reply:
Merged now.